design/media_types.go

```diff
@@ -31,3 +31,66 @@ var AuthToken = MediaType("application/vnd.authtoken+json", func() {
 		Attribute("token")
 	})
 })
+
+var WorkItem = MediaType("application/vnd.workitem+json", func() {
+	TypeName("WorkItem")
+	Description("ALM Work Item")
+	Attribute("id", String, "unique id per installation")
```

Comment:
okey, but will that not always be something that one need to handle ? by either merging or renaming on copy/move ?